src/app/views/data/data.js

```diff
@@ -20,6 +20,9 @@ class DataViewController extends ADBMapController {
     this.$log.debug('soum code:', this.soumCode);
 
     this.soumData.geojson(this.soumCode).then(geojson => this._addGeoJSONLayer(geojson));
+    this.soumData.load(this.soumCode).then(soum => {
```

Comment:
Gotcha. Yeah looks like you could do `// eslint-disable-line no-return-assign` on that line?